proposals/contrib-guide-high-level.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Might be cool if we could add a little color/visualization of the expected labels for the different steps.